gtfs/spec/en/reference.md

```diff
@@ -388,26 +390,26 @@ In regions that have multiple official languages, transit agencies/operators typ
+|  `record_sub_id` | ID | **Conditionally Required** | Helps the record that contains the field to be translated when the table doesn’t have a unique ID. Therefore, the value in `record_sub_id` is the secondary ID of the table, as defined by the table below:<br><br>- None for `agency.txt`;<br>- None for `stops.txt`;<br>- None for `routes.txt`;<br>- None for `trips.txt`;<br>- `stop_sequence` for `stop_times.txt`;<br>- None for `pathways.txt`;<br>- None for `levels.txt`;<br>- None for `attributions.txt`.<br><br>Fields in tables not defined above should not be translated. However producers sometimes add extra fields that are outside the official specification and these unofficial fields may be translated. Below is the recommended way to use `record_sub_id` for those tables:<br><br>- None for `calendar.txt`;<br>- `date` for `calendar_dates.txt`;<br>- None for `fare_attributes.txt`;<br>- `route_id` for `fare_rules.txt`;<br>- None for `shapes.txt`;<br>- `start_time` for `frequencies.txt`;<br>- `to_stop_id` for `transfers.txt`.<br><br>Conditionally Required:<br>- **Forbidden** if `table_name` is `feed_info`.<br>- **Forbidden** if `field_value` is defined.<br>- **Required** if `table_name=stop_times` and `record_id` is defined. |
+|  `field_value` | Text or URL or Email or Phone number | **Conditionally Required** | Instead of defining which record should be translated by using `record_id` and `record_sub_id`, this field can be used to define the value which should be translated. When used, the translation will be applied when the fields identified by `table_name` and `field_name` contains the exact same value defined in field_value.<br><br>The field must have **exactly** the value defined in `field_value`. If only a subset of the value matches `field_value`, the translation won’t be applied.<br><br>If two translation rules match the same record (one with `field_value`, and the other one with `record_id`), the rule with `record_id` takes precedence.<br><br>Conditionally Required:<br>- **Forbidden** if `table_name` is `feed_info`.<br>- **Forbidden** if `record_id` is defined.<br>- **Required** if `record_id` is empty. |
 
 ### feed_info.txt
 
 File: **Optional** (**Required** if `translations.txt` is provided)
 
-The file contains information about the dataset itself, rather than the services that the dataset describes. Note that, in some cases, the publisher of the dataset is a different entity than any of the agencies.
+The file contains information about the dataset itself, rather than the services that the dataset describes. Note that, in some cases, the publisher of the dataset can be a different entity than any of the agencies.
```

Comment:
I find this reads better with the "is". This is not really related to RFC 2119 but if we're editing for clarity I'd keep "is" and remove the superfluous "Note that":
```suggestion
The file contains information about the dataset itself, rather than the services that the dataset describes. In some cases, the publisher of the dataset is a different entity than any of the agencies.
```